Address safer C++ static analysis warnings in WKObject.h

cdumez opened the pull request:
#### a40f3a3fba44cd2c0d2afcde46fe1b32b1a8fe35
<pre>
Address safer C++ static analysis warnings in WKObject.h
<a href="https://bugs.webkit.org/show_bug.cgi?id=288056">https://bugs.webkit.org/show_bug.cgi?id=288056</a>

Reviewed by Timothy Hatcher and Darin Adler.

- Replace API::ObjectStorage with a new WTF::AlignedStorage type
- WTF::AlignedStorage offers several benefits since it is strongly typed and it
  doesn&apos;t rely on std::aligned_storage which is deprecated in C++23. Instead,
  it uses `alignas(Alignment) std::byte m_storage[sizeof(T)];` which is the
  suggested replacement in the standard. WTF::AlignedStorage also has the
  benefit of hiding away the reinterpret_cast which is still required and
  requirements suppression to silence static analysis
- The PR adopts WTF::AlignedStorage instead of std::aligned_storage in several
  places too.

* Source/JavaScriptCore/dfg/DFGAbstractValue.cpp:
(JSC::DFG::AbstractValue::ensureCanInitializeWithZeros):
* Source/WTF/WTF.xcodeproj/project.pbxproj:
* Source/WTF/wtf/AlignedStorage.h: Copied from Source/WebKit/UIProcess/API/Cocoa/_WKTargetedElementInfoInternal.h.
(WTF::sizeof):
* Source/WTF/wtf/HashTable.h:
(WTF::shouldValidateKey&gt;::checkKey):
* Source/WTF/wtf/NeverDestroyed.h:
(WTF::NeverDestroyed::storagePointer const):
(WTF::LazyNeverDestroyed::storagePointerWithoutAccessCheck const):
* Source/WTF/wtf/RobinHoodHashTable.h:
(WTF::SizePolicy&gt;::checkKey):
* Source/WTF/wtf/threads/Signals.h:
* Source/WebKit/SaferCPPExpectations/MemoryUnsafeCastCheckerExpectations:
* Source/WebKit/Shared/API/Cocoa/_WKFrameHandleInternal.h:
* Source/WebKit/Shared/API/Cocoa/_WKHitTestResultInternal.h:
* Source/WebKit/Shared/Cocoa/WKNSArray.mm:
* Source/WebKit/Shared/Cocoa/WKNSData.mm:
* Source/WebKit/Shared/Cocoa/WKNSDictionary.mm:
* Source/WebKit/Shared/Cocoa/WKNSNumber.mm:
* Source/WebKit/Shared/Cocoa/WKObject.h:
(API::ObjectStorage::get): Deleted.
(API::ObjectStorage::operator*): Deleted.
(API::ObjectStorage::operator-&gt;): Deleted.
* Source/WebKit/UIProcess/API/Cocoa/WKBackForwardList.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKBackForwardListItem.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKContentRuleListInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKContentRuleListStoreInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKContentWorldInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKContextMenuElementInfoInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKDownloadInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKFrameInfoInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKHTTPCookieStoreInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKNavigation.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKNavigationActionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKNavigationData.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKNavigationResponseInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKOpenPanelParametersInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKPreferencesInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKProcessPoolInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKSecurityOriginInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKURLSchemeTaskInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKUserContentControllerInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKUserScriptInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionActionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionCommandInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionContextInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionControllerConfigurationInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionControllerInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionDataRecordInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionMatchPatternInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionMessagePortInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfigurationInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebpagePreferencesInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataRecordInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStoreInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWindowFeaturesInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKApplicationManifestInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKAttachmentInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKAutomationSessionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKContentRuleListActionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKContextMenuElementInfoInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKCustomHeaderFieldsInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKDataTaskInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKFeatureInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKFrameTreeNodeInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKGeolocationPositionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKInspectorConfigurationInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKInspectorDebuggableInfoInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKInspectorExtensionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKInspectorInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKProcessPoolConfigurationInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKResourceLoadInfoInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKResourceLoadStatisticsFirstPartyInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKResourceLoadStatisticsThirdPartyInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKTargetedElementInfoInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKTargetedElementRequestInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKTextRunInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKUserInitiatedActionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKUserStyleSheetInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKVisitedLinkStoreInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationAssertionResponseInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanelInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionSidebarInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebPushDaemonConnectionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebPushMessageInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebPushSubscriptionDataInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebsiteDataStoreConfigurationInternal.h:
* Source/WebKit/WebProcess/InjectedBundle/API/Cocoa/WKWebProcessPlugInCSSStyleDeclarationHandle.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/Cocoa/WKWebProcessPlugInFrame.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/Cocoa/WKWebProcessPlugInHitTestResult.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/Cocoa/WKWebProcessPlugInNodeHandle.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/Cocoa/WKWebProcessPlugInRangeHandle.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/Cocoa/WKWebProcessPlugInScriptWorld.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/mac/WKWebProcessPlugIn.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/mac/WKWebProcessPlugInBrowserContextController.mm:

Canonical link: <a href="https://commits.webkit.org/290792@main">https://commits.webkit.org/290792@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/15be64f93f2a8ffc30ae437e660f2648d909976b

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/91086 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/10629 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/148 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/96094 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/41856 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/11038 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/18949 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/70008 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/27535 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/94087 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/8420 "Passed tests") | [❌ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/82556 "Found 1 new API test failure: TestWebKitAPI.WKWebExtensionAPIDevTools.PortMessagePassingToBackground (failure)") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/50334 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/8191 "Passed tests") | [❌ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/125 "Found 60 new test failures: animations/font-variations/font-stretch.html fast/css/aspect-ratio-min-height-replaced.html fast/workers/worker-page-cache.html http/tests/loading/main-resource-delegates-on-back-navigation.html http/tests/loading/unfinished-load-back-to-cached-page-callbacks.html http/tests/media/hls/track-in-band-ext-x-date-range-start-time.html http/tests/security/clean-origin-css-exposed-resource-timing.html http/tests/security/contentSecurityPolicy/block-all-mixed-content/insecure-script-in-iframe.html http/tests/security/contentSecurityPolicy/script-src-none-inline-event.html http/tests/security/contentSecurityPolicy/script-src-parsing-implicit-and-explicit-port-number.html ... (failure)") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/40986 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/83907 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/78513 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/142 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/98072 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/89856 "Built successfully and passed tests") | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/18293 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/13415 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/79019 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/18552 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/78384 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/78220 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/22723 "Passed tests") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/137/builds/93 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/11502 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/14385 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/18294 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/23626 "Built successfully") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/112424 "Built successfully") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/18020 "Built successfully") | | [❌ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/32647 "Found 13 new JSC stress test failures: microbenchmarks/memcpy-wasm-medium.js.mini-mode, microbenchmarks/memcpy-wasm-small.js.bytecode-cache, microbenchmarks/memcpy-wasm-small.js.mini-mode, microbenchmarks/memcpy-wasm-small.js.no-llint, microbenchmarks/memcpy-wasm.js.no-llint, wasm.yaml/wasm/function-tests/memcpy-wasm-loop.js.wasm-slow-memory, wasm.yaml/wasm/stress/b3-signed-extend-16-to-64.js.default-wasm, wasm.yaml/wasm/stress/ipint-bbq-osr-with-try5.js.default-wasm, wasm.yaml/wasm/stress/ipint-bbq-osr-with-try5.js.wasm-eager-jettison, wasm.yaml/wasm/stress/live-funcref-across-loop-tier-up.js.wasm-collect-continuously ... (failure)") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/21481 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/19799 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

darinadler commented on Source/WebKit/Shared/Cocoa/WKObject.h:
```diff
@@ -28,6 +28,7 @@
 #import "WKFoundation.h"
 
 #import <type_traits>
+#import <wtf/AlignedStorage.h>
```
Why do we need this include in this header? I'd expect it to be in other headers and maybe some implementation files.

cdumez replied:
It was either that or adding the include to every file that was using API::ObjectStorage (and there were a lot). I took the easy way out and since all these files had to already include WKObject.h, I just added the include here.

I can go the hard way if you prefer though.

darinadler replied:
I do prefer it, but not strongly enough to insist.

cdumez replied:
I did it before landing.

cdumez committed:
Address safer C++ static analysis warnings in WKObject.h
https://bugs.webkit.org/show_bug.cgi?id=288056

Reviewed by Timothy Hatcher and Darin Adler.

- Replace API::ObjectStorage with a new WTF::AlignedStorage type
- WTF::AlignedStorage offers several benefits since it is strongly typed and it
  doesn't rely on std::aligned_storage which is deprecated in C++23. Instead,
  it uses `alignas(Alignment) std::byte m_storage[sizeof(T)];` which is the
  suggested replacement in the standard. WTF::AlignedStorage also has the
  benefit of hiding away the reinterpret_cast which is still required and
  requirements suppression to silence static analysis
- The PR adopts WTF::AlignedStorage instead of std::aligned_storage in several
  places too.

* Source/JavaScriptCore/dfg/DFGAbstractValue.cpp:
(JSC::DFG::AbstractValue::ensureCanInitializeWithZeros):
* Source/WTF/WTF.xcodeproj/project.pbxproj:
* Source/WTF/wtf/AlignedStorage.h: Copied from Source/WebKit/UIProcess/API/Cocoa/_WKTargetedElementInfoInternal.h.
(WTF::sizeof):
* Source/WTF/wtf/HashTable.h:
(WTF::shouldValidateKey>::checkKey):
* Source/WTF/wtf/NeverDestroyed.h:
(WTF::NeverDestroyed::storagePointer const):
(WTF::LazyNeverDestroyed::storagePointerWithoutAccessCheck const):
* Source/WTF/wtf/RobinHoodHashTable.h:
(WTF::SizePolicy>::checkKey):
* Source/WTF/wtf/threads/Signals.h:
* Source/WebKit/SaferCPPExpectations/MemoryUnsafeCastCheckerExpectations:
* Source/WebKit/Shared/API/Cocoa/_WKFrameHandleInternal.h:
* Source/WebKit/Shared/API/Cocoa/_WKHitTestResultInternal.h:
* Source/WebKit/Shared/Cocoa/WKNSArray.mm:
* Source/WebKit/Shared/Cocoa/WKNSData.mm:
* Source/WebKit/Shared/Cocoa/WKNSDictionary.mm:
* Source/WebKit/Shared/Cocoa/WKNSNumber.mm:
* Source/WebKit/Shared/Cocoa/WKObject.h:
(API::ObjectStorage::get): Deleted.
(API::ObjectStorage::operator*): Deleted.
(API::ObjectStorage::operator->): Deleted.
* Source/WebKit/UIProcess/API/Cocoa/WKBackForwardList.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKBackForwardListItem.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKContentRuleListInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKContentRuleListStoreInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKContentWorldInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKContextMenuElementInfoInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKDownloadInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKFrameInfoInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKHTTPCookieStoreInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKNavigation.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKNavigationActionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKNavigationData.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKNavigationResponseInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKOpenPanelParametersInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKPreferencesInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKProcessPoolInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKSecurityOriginInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKURLSchemeTaskInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKUserContentControllerInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKUserScriptInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionActionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionCommandInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionContextInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionControllerConfigurationInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionControllerInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionDataRecordInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionMatchPatternInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionMessagePortInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfigurationInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebpagePreferencesInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataRecordInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStoreInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/WKWindowFeaturesInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKApplicationManifestInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKAttachmentInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKAutomationSessionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKContentRuleListActionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKContextMenuElementInfoInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKCustomHeaderFieldsInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKDataTaskInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKFeatureInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKFrameTreeNodeInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKGeolocationPositionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKInspectorConfigurationInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKInspectorDebuggableInfoInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKInspectorExtensionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKInspectorInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKProcessPoolConfigurationInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKResourceLoadInfoInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKResourceLoadStatisticsFirstPartyInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKResourceLoadStatisticsThirdPartyInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKTargetedElementInfoInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKTargetedElementRequestInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKTextRunInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKUserInitiatedActionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKUserStyleSheetInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKVisitedLinkStoreInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationAssertionResponseInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanelInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionSidebarInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebPushDaemonConnectionInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebPushMessageInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebPushSubscriptionDataInternal.h:
* Source/WebKit/UIProcess/API/Cocoa/_WKWebsiteDataStoreConfigurationInternal.h:
* Source/WebKit/WebProcess/InjectedBundle/API/Cocoa/WKWebProcessPlugInCSSStyleDeclarationHandle.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/Cocoa/WKWebProcessPlugInFrame.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/Cocoa/WKWebProcessPlugInHitTestResult.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/Cocoa/WKWebProcessPlugInNodeHandle.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/Cocoa/WKWebProcessPlugInRangeHandle.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/Cocoa/WKWebProcessPlugInScriptWorld.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/mac/WKWebProcessPlugIn.mm:
* Source/WebKit/WebProcess/InjectedBundle/API/mac/WKWebProcessPlugInBrowserContextController.mm:

Canonical link: https://commits.webkit.org/290792@main
